### DISCUSSION TITLE
Anyone have a Tentacle T3 failure yet?

### DISCUSSION THREAD

**Discussion by johnny2678** [tagyoureit/nodejs-poolController#943] [Q&A] [answered]
Woke up yesterday to stagnant ph/orp probe readings.  Confirmed in REM that readings weren't updating.

pH probe chip is blinking red and errors in log indicate that values from the probe chips are invalid?
```
1|REM  | [03/05/2024, 10:51:47] debug: 39 - Multi-Relay Board Executed read command 0x00 byte read:0x00
1|REM  | [03/05/2024, 10:51:48] debug: 39 - Multi-Relay Board Executed read command 0x03 byte read:0x00
1|REM  | [03/05/2024, 10:51:48] debug: 39 - Multi-Relay Board Executed read command 0x00 byte read:0x00
1|REM  | [03/05/2024, 10:51:48] debug: 39 - Multi-Relay Board Executed read command 0x03 byte read:0x00
1|REM  | [03/05/2024, 10:51:48] debug: 39 - Multi-Relay Board Executed read command 0x00 byte read:0x00
1|REM  | [03/05/2024, 10:51:48] warn: Atlas_EZO-ORP could not retrieve status result was empty
1|REM  | [03/05/2024, 10:51:48] warn:  could not retrieve status result was empty
1|REM  | [03/05/2024, 10:51:48] error: Atlas_EZO-ORP Discarded invalid value from I2C result:undefined
1|REM  | [03/05/2024, 10:51:48] verbose: Feed evaluating undefined: 536.5 samples [536.5,536.5,536.5,536.5,536.5]
1|REM  | [03/05/2024, 10:51:48] silly: Sending on MQTT Channel {"eventName":"pool/chem/orp","value":536.5}
1|REM  | [03/05/2024, 10:51:48] verbose: Feed mqttClient/15620 MQTT sending orp: 536.5 to pool/chem/orp
1|REM  | [03/05/2024, 10:51:48] verbose: Feed evaluating orpLevel: 536.5 samples [536.5,536.5,536.5,536.5,536.5]
1|REM  | [03/05/2024, 10:51:48] verbose: Emitting: /chemController : {"orpLevel":536.5,"id":50}
1|REM  | [03/05/2024, 10:51:48] verbose: Feed njspc/njspc-15620 sending orpLevel: 536.5 to i2c:1:1
1|REM  | [03/05/2024, 10:51:48] error:  Discarded invalid value from I2C result:undefined
1|REM  | [03/05/2024, 10:51:48] verbose: Feed evaluating undefined: 7.616 samples [7.616,7.616,7.616,7.616,7.616]
1|REM  | [03/05/2024, 10:51:48] silly: Sending on MQTT Channel {"eventName":"pool/chem/ph","value":7.616}
```

`i2cdetect -y 1` shows that probes aren't even detected anymore?
![image](https://github.com/tagyoureit/nodejs-poolController/assets/8219743/985ba142-0425-4e64-b820-ff29eb3235c5)

was looking for 98/99 but all I see is the sequent smart fan, relay boards, and mega-bas

**Comment by rstrouse** [comment]
No but I have had a mis-behaving device disrupt the i2c channel.  On the EZO boards do the leds ever show as green?  They should only ever show as blue.  If you see a green I suspect that they reverted to uart.

**Reply by johnny2678** [reply]
yep, they are blue except when taking readings.  The ORP chip flashes green then back to blue.  The pH chip flashes red, then back to blue.  Neither are sending readings to REM.

Didn't these used to be more $$$ - I guess at this price I could try another one? -https://www.whiteboxes.ch/shop/whitebox-t3-for-raspberry-pi/

**Reply by johnny2678** [reply]
or... I guess it could be the sensor chip(s)? but what are the chances they both go out in the same day?  I was out of town weds when it went out, so it's not from me tinkering! 😂

**Comment by johnny2678** [comment, marked as answer]
and.... disregard!  Something got stuck so tried a full power cycle of the pi stack.  OS Shutdown - pull power for 30s - reconnect power.

Everything was there when I brought it back up.  sorry for the fire drill!

**Comment by johnny2678** [comment]
well, I just ordered a replacement T3 this AM. 

Woke up to static PH and ORP readings and this time a hard power cycle didn't restore the probes.  

```
1|REM  | [23/06/2024, 08:46:10] error: Atlas_EZO-ORP Discarded invalid value from I2C result:undefined
1|REM  | [23/06/2024, 08:46:10] error:  Discarded invalid value from I2C result:undefined
1|REM  | [23/06/2024, 08:46:11] warn: Atlas_EZO-ORP could no:t retrieve status result was empty
1|REM  | [23/06/2024, 08:46:11] warn:  could not retrieve status result was empty
```

For some reason they aren't showing up in i2cdetect either :
```
pi@PiPoolv2:~/AtlasScientificI2C/src $ sudo i2cdetect -y 1
     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
00:          03 -- -- -- -- -- -- -- -- -- -- -- -- 
10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
20: -- -- -- -- -- -- -- 27 -- -- -- -- -- -- -- -- 
30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
40: -- -- -- -- -- -- -- -- 48 -- -- -- -- -- -- -- 
50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
70: -- -- -- -- -- -- -- -- 
```

But my smart fan, relay board, and mega-bas are still detected.  So in my head it's down to the T3 or the probe chips.  At $37, I ordered the upgraded T3 just in case.  Happy to hear any more ideas if you have them.

**Reply by FlaMike** [reply]
Dude!  Both of us seem to be having a challenging pool equipment year (ORP probes, controllers and more!).  So sorry to hear about your most recent one.  I probably bought the last old school T-3s in December. , for around $100, if I recall correctly.  I wonder if A-S is cost-reducing their entire lineup along with Sequent (think smart fans) & just messing with us as a result.

My gold ORP probe seems to be working much better than the platinum one, but now I am struggling to locate a suction side air leak & 3 flaky & crappy Pentair GloBrite lights.  I'm trying to get Jandy Hydrocool Intellibright-compatible replacements & it's proving to be a bear to source them.  Oh what a world!

Good luck with your T3 struggle.  I feel for you!!!

**Reply by johnny2678** [reply]
The sad thing is my cheap consumer probes were finally working flawlessly prior to Friday afternoon.  Then both just were dropped from i2c simultaneously.  Probe sensor chips lights are both blue and don't blink at all.  It's like the T3 card isn't even there.

Hope the new one gets me back up and running.

Crazy that the price dropped from >$100 to $35.  Don't need the BNC adapters on the new one either.

Good luck with yours as well!  I've been purposefully putting off reading your Easytouch -> Nixie write up.  I need that project like I need a hole in my head but it's always something I wanted to do.  Thanks for writing that!

**Reply by FlaMike** [reply]
I have a lot more writing to do regarding my "conversion" from ET to Nixie.  So far, all I've done is the background & motivation.  I need to get the next chapter together... overall design & rationale.  So many lessons learned in the process I should be a genius by now.  Alas, I'm still the same guy I was before I dove in.  Just a bit more battered & bruised, but still alive & kicking albeit with less power & height :-)
